tsconfig.json

```diff
@@ -30,6 +30,7 @@
       "@app-globals": ["src/app-globals/index.ts"],
       "@compiler-deps": ["src/compiler/sys/modules/compiler-deps.ts"],
       "@dev-server-process": ["src/dev-server/server-process.ts"],
+      "@environment": ["src/compiler/sys/environment.ts"],
```

Comment:
I believe that we programmatically insert an import to this module into TypeScript during the rollup build (so an import from `@environment` doesn't appear anywhere in our code) but if we want it to resolve correctly the easiest / best thing to do is just add it here to our `paths`. We could also then start using this alias throughout our code if we want to.